com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/UpsampleTransparent.shader

```diff
@@ -40,7 +40,7 @@ Shader "Hidden/HDRP/UpsampleTransparent"
 #define DEBUG_EDGE 0
 #endif
 
-
+#if SHADER_STAGE_FRAGMENT
```

Comment:
This is not needed anymore, I fixed this in master to have BILINEAR be the default when nothing is defined.